run_xnat2bids.py

```diff
@@ -497,49 +570,55 @@ async def main():
     # Instantiate argument parser
     args = parse_cli_arguments()
 
-    # Load default config file into dictionary
-    script_dir = pathlib.Path(__file__).parent.resolve()
-    default_params = load(f'{script_dir}/x2b_default_config.toml')
-
-    # Set arg_dict. If user provides config, merge dictionaries.
-    arg_dict = merge_default_params(args.config, default_params)
-        
     # Fetch user credentials 
     user, password = get_user_credentials()
 
-    # Initialize bids_root for non-local use
-    bids_root = f"/users/{user}/bids-export/"
+    if (args.diff):
+        if (args.bids_root) and (os.path.exists(args.bids_root)):
```

Comment:
It would be helpful to throw an error here if the participant didn't pass a bids_root directory, or they did but it doesn't exist.

Reply:
Typer will verify bids_root is present and bark otherwise. I will add some logging in the case where bids_root does not exist. Thanks!